src/TouristInfoCards.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import "./TouristInfoCards.css";
+
+const Card = ({ image, title, description, link }) => {
+  return (
+    <div className="card">
+      <img src={image} className="card-img-top" alt={title} />
+      <div className="card-body">
+        <h5 className="card-title">{title}</h5>
+        <p className="card-text">{description}</p>
+        <a href={link} className="btn btn-primary">
+          More Information
+        </a>
+      </div>
+    </div>
+  );
+};
+
+function TouristInfoCards() {
```

Comment:
I would personally stick to one way to declare react functional components, so this is more of a consistency/nitpick comment

https://github.com/CodeYourFuture/cyf-hotel-react/pull/573#discussion_r1215415298